stories/cosmoz-input.stories.js

```diff
@@ -60,9 +60,8 @@ export const contour = () => html`
 			--cosmoz-input-contour-size: 1px;
 			--cosmoz-input-label-translate-y: 10px;
 			--cosmoz-input-float-display: none;
-		}
-		cosmoz-input:not(:first-child) {
-			margin-top: 10px;
+			--cosmoz-input-padding-top: 10px;
+			--cosmoz-input-padding-bottom: 10px;
```

Comment:
I would prefer to drop these two and instead set `--cosmoz-input-padding: 10px 8px;`

Reply:
Yes. One padding property is enough. The properties for padding-bottom and padding-top are more like a hack to have the same variables in paper-input and cosmoz-input.

Reply:
This story still sets `--cosmoz-input-padding-top` and `--cosmoz-input-padding-bottom`